src/instructlab/model/serve_backend.py

```diff
@@ -80,11 +114,25 @@ def serve_backend(
     elif backend == backends.VLLM:
         # First Party
         from instructlab.cli.model.serve import warn_for_unsupported_backend_param
-        from instructlab.utils import contains_argument
 
         warn_for_unsupported_backend_param(ctx)
 
         ctx.obj.config.serve.vllm.vllm_args = ctx.obj.config.serve.vllm.vllm_args or []
+
+        # Fix ValueError: Too large swap space. 16.00 GiB out of the 15.01 GiB total CPU memory is allocated for the swap space.
```

Comment:
I am guessing this message is for a specific use case, might be good to write the scenario it's handling more generically.